examples/wasm-cc/envoy.yaml

```diff
@@ -22,28 +22,23 @@ static_resources:
                   prefix: "/"
                 route:
                   cluster: web_service
+
           http_filters:
           - name: envoy.filters.http.wasm
             typed_config:
-              "@type": type.googleapis.com/udpa.type.v1.TypedStruct
-              type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
-              value:
-                config:
-                  name: "my_plugin"
-                  root_id: "my_root_id"
-                  configuration:
-                    "@type": "type.googleapis.com/google.protobuf.StringValue"
-                    value: |
-                      {}
```

Comment:
While the configuration is empty in this example, it shows how to configure plugins, so I think there is some value that's being removed here. Could you re-add it?

If you don't want to show an empty configuration, then maybe update the example Wasm filter to inject configured value instead of `BAR`?

Reply:
yep - i can readd - i was ambivalent about removing - i did so for simplicity and because there wasnt any explanation as to how its used

Reply:
ill add a comment...

Reply:
re/added